### PR TITLE
Added Enumerable helper Count

### PR DESCRIPTION
**Pull request by epDugas**
Added Enumerable Helper Count and associated test.

### CODE REVIEW

File: src/Handlebars.Net.Helpers/Helpers/EnumerableHelpers.cs [hunk 1]
```diff
@@ -58,6 +58,12 @@ public bool IsEmpty(IEnumerable<object?>? value)
         return value is null || !value.Any();
     }
 
+    [HandlebarsWriter(WriterType.Value)]
```
Comment (StefH) [inline src/Handlebars.Net.Helpers/Helpers/EnumerableHelpers.cs:61]: Can you place this method alphabetically sorted, so before Distinct ?

**Review by StefH — CHANGES_REQUESTED** [review]
Thanks for the PR.

I see that there is no "real" Enumerable template test yet.

Can you add a new test, just like BooleanHelpersTemplateTests ?
(You only need to add "Count" as a test : valid and null)

**Review by StefH — APPROVED** [review]
